ui/src/config/section/compute.js

```diff
@@ -207,9 +207,10 @@ export default {
           docHelp: 'adminguide/virtual_machines.html#virtual-machine-snapshots',
           dataView: true,
           popup: true,
-          show: (record) => {
-            return ((['Running'].includes(record.state) && record.hypervisor !== 'LXC') ||
-              (['Stopped'].includes(record.state) && !['KVM', 'LXC'].includes(record.hypervisor)))
+          show: (record, store) => {
+            return (record.hypervisor !== 'KVM') ||
```

Comment:
why destroyed state?

Reply:
I tried to keep the UI conditions same as the service layer. Please see [hostSupportsSnapsthotForVolume()](https://github.com/apache/cloudstack/blob/main/server/src/main/java/com/cloud/storage/snapshot/SnapshotManagerImpl.java#L1344).

But I'm ok if we don't want to allow snapshot creation from UI for destroyed VMs.